resources/views/navigation-menu.blade.php

```diff
@@ -20,123 +26,72 @@
 ];
 @endphp
 
-
-
-<nav x-data="{ open: false }">
-    <div class="px-4 mx-auto max-w-7xl sm:px-6 lg:px-8 navbar">
-        <div class="flex py-1 justify-evenly ">
-            <div class="hidden space-x-8 sm:-my-px sm:ml-10 sm:flex">
-                <x-jet-nav-link :href="route('home')" :active="request()->routeIs('home')">
-                    Accueil
+<nav x-data="{ mobileOpen: false }" class="mx-auto px-8 block border-none min-h-0 relative container md:max-w-screen-lg navbar">
+    <div class="flex py-1 justify-between">
+        <div>
+            <x-jet-nav-link :href="route('home')" :active="request()->routeIs('home')">
+                <img id="rc-logo-homepage" src="{{ asset('images/inline_org_color.png') }}">
+            </x-jet-nav-link>
+        </div>
+        <div class="space-x-8 px-4 hidden <?= $mobileSizeTriggerFlex ?>">
```

Comment:
Don't use `<?= ... ?>`, use `{{ ... }}` instead but I don't see why you need PHP variables here in the first place, it has nothing to do with tailwind